MapboxNavigationTests/SimulatedLocationManagerTests.swift

```diff
@@ -13,7 +13,15 @@ class SimulatedLocationManagerTests: FBSnapshotTestCase {
     }
 
     func testSimulateRouteDoublesBack() {
-        let route = Fixture.routesFromMatches(at: "sthlm-double-back")![0]
+        let routeExpectation = expectation(description: "Gets a matching route")
+        var route: Route!
+        Fixture.getRoutesFromMatches(at: "sthlm-double-back") { (waypoints, routes, error) in
+            route = routes?[0]
```

Comment:
Nit: Can we use `routes.first` here? Subscript accessors give me the willies.

Reply:
I can make that change as part of #1759, but note that it’s being set to a `Route!`-typed variable anyways. 😉